app/controllers/holidays_controller.rb

```diff
@@ -80,19 +83,20 @@ def create_params
   end
 
   def batch_create_params
-    params
-      .require(:holiday)
-      .permit(
-        :title,
-        :all_day,
-        :start_at,
-        :end_at
-      )
-      .merge(users: user_ids)
+    params.require(:holiday)
```

Comment:
What's the rationale for changing the indentation style here? Can we keep the original style instead? It's consistent.